web-common/src/features/entity-management/resource-selectors.ts

```diff
@@ -112,13 +113,18 @@ export function useResourceV2<T = V1Resource>(
   );
 }
 
-export function useProjectParser(queryClient: QueryClient, instanceId: string) {
+export function useProjectParser(
+  queryClient: QueryClient,
+  instanceId: string,
+  enabled: boolean = true,
+) {
   return useResource(
     instanceId,
     SingletonProjectParserName,
     ResourceKind.ProjectParser,
     {
       queryClient,
+      enabled,
     },
   );
 }
```

Comment:
It'd be nice to change the function signature to accept a `queryOptions` object, then we won't need to keep modifying it to accept more individual `queryOptions` going forward. See [this instance of `useDashboard`](https://github.com/rilldata/rill/blob/cyrus/prevent-requests-no-measures/web-common/src/features/dashboards/selectors.ts#L25-L40).